File: beginner/javascript/1115.js
var input = require('fs').readFileSync('stdin', 'utf8');

var linha = input.split('\n');

var i = -1;

for(;;){
    i++;
    valores = linha[i].split(' ').map(num => parseInt(num));

    if(valores[0] > 0 && valores[1] > 0)
        console.log("primeiro");
    else if(valores[0] > 0 && valores[1] < 0)
        console.log("quarto");
    else if(valores[0] < 0 && valores[1] < 0)
        console.log("terceiro");
    else if(valores[0] < 0 && valores[1] > 0)
        console.log("segundo");
    else if(valores[0] == 0 || valores[1] == 0)
        break;
}

File: beginner/java/1115.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Locale.setDefault(Locale.US);

        Scanner scanner = new Scanner(System.in);

        int x, y;

        for(;;){
            x = scanner.nextInt();
            y = scanner.nextInt();

            if(x > 0 && y > 0)
                System.out.println("primeiro");
            else if(x > 0 && y < 0)
                System.out.println("quarto");
            else if(x < 0 && y < 0)
                System.out.println("terceiro");
            else if(x < 0 && y > 0)
                System.out.println("segundo");
            else if(x == 0 || y == 0)
                break;
        }

        scanner.close();

    }
}

